java/com/google/copybara/git/GitEnvironment.java

```diff
@@ -36,11 +35,17 @@ public GitEnvironment(Map<String, String> environment) {
   }
 
   public ImmutableMap<String, String> getEnvironment() {
-    Map<String, String> env = Maps.newHashMap(environment);
+    ImmutableMap.Builder<String, String> env = ImmutableMap.builder();
+    env.putAll(environment);
+
+    // Explicitly set output language to english so parsing of git's output
+    // succeeds independently of users default locale.
+    env.put("LANG", "en_US.UTF-8");
```

Comment:
Actually, this doesn't work because this is an immutable map and cannot have duplicates.  Sorry, I should've spot this in the review.

You can see the error by running bazel //... on your change.

The fix is be to leave everything as it was before and just add the put statement like we do for `GIT_TERMINAL_PROMPT`. We make an immutable copy at the end anyway.

Reply:
Also note that you'll need to update `GitEnvironmentTest.java`.